addon/-private/column-tree.js

```diff
@@ -630,8 +630,10 @@ export default EmberObject.extend({
     }
 
     let containerWidthAdjustment = get(this, 'containerWidthAdjustment') || 0;
+    let containerScrollbarWidth = getScrollbarWidth(this.container);
+    let containerInnerWidth = getInnerClientRect(this.container).width;
     let containerWidth =
-      getInnerClientRect(this.container).width * this.scale + containerWidthAdjustment;
+      (containerInnerWidth - containerScrollbarWidth) * Number(Math.round(this.scale+'e2')+'e-2') + containerWidthAdjustment;
```

Comment:
I do not fully understand the purpose of `this.scale`.
The problem is that the `getScale` function compare `container.offsetHeight` and `container.getBoundingClientRect().height`, depending on the browser/window size, those values can be computed as integers or float.
So in some cases, you could have a float value for `container.getBoundingClientRect().height` and an integer value for `container.offsetHeight` which result on a scale not accurate.

![2019-09-11-142146_1364x237_scrot](https://user-images.githubusercontent.com/1716173/64705458-83f0dd80-d49f-11e9-94f8-667b825270f7.png)